packages/editor-ui/src/store.ts

```diff
@@ -60,6 +63,7 @@ const state: IRootState = {
 	activeCredentialType: null,
 	// @ts-ignore
 	baseUrl: import.meta.env.VUE_APP_URL_BASE_API ? import.meta.env.VUE_APP_URL_BASE_API : (window.BASE_PATH === '/{{BASE_PATH}}/' ? '/' : window.BASE_PATH),
+	currentWorkflowExecutions: [],
```

Comment:
can we move this instead to a module instead and follow the patterns there? not sure which module is better.. but would be better not to make store bigger as we try to break it apart and this seems self contained

Reply:
My thought initially, but I wasn't sure where should it go. Ok, moving it to `workflows`, that seems to be a good home for all workflow-related stuff for the future.